pages/api/index.js

```diff
@@ -10,6 +12,14 @@ function API({url, content, locale}) {
       locale={locale}
       contributors={content.contributors}
     >
+      <Head>
+        {twitterCard({
+          card: 'summary',
+          title: content.title,
```

Comment:
Yup, sure 👍 The `title` was implemented like `glamorous - React component styling solved 💄`. So I'll make the changes to match like in the `title`.

E.g. `glamorous - Basic Concepts`